src/forms/Typeahead.jsx

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Downshift from 'downshift';
+import cx from 'classnames';
+
+import TextInput from './TextInput';
+
+export const TA_DROPDOWN_CLASSNAME = 'typeahead-dropdown';
+export const TA_ITEM_CLASSNAME = 'typeahead-item';
+
+/**
+ * @module Typeahead
+ */
+class Typeahead extends React.PureComponent {
+
+	render() {
+		const {
+			inputProps,
+			height,
+			items,
+			...other
+		} = this.props;
+
+		return (
+			<Downshift {...other}>
+				{({
+					getInputProps,
+					getItemProps,
+					isOpen,
+					highlightedIndex,
+				}) =>
+				(<div className="typeahead">
+					<TextInput
+						className="typeahead-input"
+						{...getInputProps({
+							...inputProps
+						})}
+					/>
+					{isOpen
+						?
+							<div
```

Comment:
Are there any props we need to pass to the dropdown element in order for `Downshift` to apply the correct aria attributes?

Reply:
`Downshift` adds its own aria attributes, but if you see any missing in the rendered HTML, let me know and I'll add them